src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -127,6 +138,7 @@ def get(self, request: Request, project: Project) -> Response:
             return Response({"detail": "Filepath is required"}, status=400)
 
         ctx = {
+            "file": request.GET.get("file"),
```

Comment:
This is in order so we can support `scope.set_tag("stacktrace_link.in_app", "site-packages" in ctx["file"])` inside of `set_top_tags`.